lib/ansible/modules/network/vyos/_vyos_l3_interface.py

```diff
@@ -33,6 +33,10 @@
 description:
   - This module provides declarative management of L3 interfaces
     on VyOS network devices.
+deprecated:
+  removed_in: '2.13'
```

Comment:
same question as in the previous commit.

Reply:
@danielmellado deprecated for now.